mne/event.py

```diff
@@ -210,6 +211,10 @@ def read_events(filename, include=None, exclude=None, mask=0):
     mask : int | None
         The value of the digital mask to apply to the stim channel values.
         The default value is 0. ``None`` skips masking.
+    mask_type: 'and' | 'not_and'
+        The type of operation between the mask and the trigger.
```

Comment:
Add a note that MNE-C behavior is `'and'`. I'm +1 on a deprecation cycle with `None -> 'not_and'` to eventually get `and` to be the default. @agramfort WDYT?